virttest/guest_agent.py

```diff
@@ -552,6 +552,50 @@ def set_vcpus(self, action):
         return self.cmd(cmd=cmd, args=action)
 
     @error_context.context_aware
+    def get_time(self):
```

Comment:
If the command is not supported for this guest-agent version ,it will raise the error in test case (tp-qemu). the error was handled there.